backend/docs/api.md

```diff
@@ -27,6 +27,8 @@ Example request:
 }
 ```
 
+## Tickets
+
 ### POST /shop/:shop/ticket/
```

Comment:
```suggestion
### POST /shop/:shop/ticket
```

I think this should be without slash as well?

Reply:
By default the express router actually doesn't care. So it's already implemented like in the docs.

In the POST request, the trailing `/` should be used. The POST creates a new resource in the container (the URL with the trailing `/`). The new resource URL is then container URL + something.

Reply:
You are way more familiar with the specs than me! I thought it needs to be consistent with the other routes, but if it's dependent on `POST` and `GET` being different (see the change to `GET /shop/:shop/timeslot`), then I'm totally okay with it. I'll merge now and will do as you say to be as compliant as possible :)